src/utils.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export function resolve (host, path) {
+  if (!host) {
+    return path;
+  }
+  const hostEndWithSlash = host[host.length - 1] === '/'
+  const pathStartsWithSlash = path[0] === '/'
+
+  if (hostEndWithSlash === true && pathStartsWithSlash === true) {
```

Comment:
could you add a comment why these cases can occur?

Since EMS always publishes paths with a slash preceding (since it's a path from root), this really is here to accommodate the use-case of the kibana-as-proxy rerouting.

a host ending with a slash should be PEBKAC and can be enforced too.